docs/modules/user-guide/partials/proc_adding-commands-to-a-devfile.adoc

```diff
@@ -54,59 +54,22 @@ components:
       mountSources: true
       command: ['sleep', 'infinity']
       env:
-        - name: GOPATH
+        - name: gopath
           value: $(PROJECTS_ROOT)/go
-        - name: GOCACHE
+        - name: gocache
```

Comment:
Can you proofcheck golang doesn't expect this environment variable to be really `GOCACHE`?